### PR TITLE
Add 2 basic examples to TON Cookbook

### PR DESCRIPTION
**Pull request by coalus**

### CODE REVIEW

**Comment by Gusarich** [comment]
@coalus why don't we use the default Jetton wrapepr from `@ton/ton` library in "How to calculate user's Jetton wallet address." example?

Like this:
```ts
const jettonMaster = provider.open(JettonMaster.create(Address.parse('address')))
console.log(await jettonMaster.getWalletAddress(Address.parse('user address')))
```

File: docs/develop/dapps/cookbook.md [hunk 1]
```diff
@@ -239,6 +239,86 @@ print(address.to_str(is_user_friendly=True, is_bounceable=False, is_url_safe=Tru
...
+    });
+    const userAddressCell = beginCell().storeAddress(userAddress).endCell()
+    
+    const response = await client.runMethod(jettonMasterAddress, "get_wallet_address", [{type: "slice", cell: userAddressCell}])
+    return response.stack.readAddress() 
+}
+const jettonMasterAddress = Address.parse('...') // for example EQBlqsm144Dq6SjbPI4jjZvA1hqTIP3CvHovbIfW_t-SCALE
+const userAddress = Address.parse('...')
+
+getUserWalletAddress(userAddress, jettonMasterAddress)
```
Comment (Gusarich) [inline docs/develop/dapps/cookbook.md:317]: add console.log here to match the structure of the first option

File: docs/develop/dapps/cookbook.md [hunk 2]
```diff
@@ -314,7 +314,7 @@ async function getUserWalletAddress(userAddress, jettonMasterAddress) {
 const jettonMasterAddress = Address.parse('...') // for example EQBlqsm144Dq6SjbPI4jjZvA1hqTIP3CvHovbIfW_t-SCALE
 const userAddress = Address.parse('...')
 
-getUserWalletAddress(userAddress, jettonMasterAddress)
+console.log(getUserWalletAddress(userAddress, jettonMasterAddress))
```
Comment (Gusarich) [inline docs/develop/dapps/cookbook.md:317]: it's async function so there should be "await"

**Comment by reveloper** [comment]
@coalus, @Gusarich Thank you!

**Pull request merged**